### PR TITLE
Update Croatian

### PR DESCRIPTION
**Pull request by milotype**

### CODE REVIEW

**Comment by milotype** [comment]
I don't understand the problem. Please explain, or correct as needed.
Thanx

File: Sparkle/hr.lproj/Sparkle.strings [hunk 1]
```diff
@@ -78,16 +78,16 @@
...
-"Should %1$@ automatically check for updates? You can always check for updates manually from the %1$@ menu." = "Da li da %1$@ automatski provjerava nadogradnje? Nadogradnje možeš i ručno provjeriti u %1$@ izborniku.";
+"Should %1$@ automatically check for updates? You can always check for updates manually from the %1$@ menu." = "Želiš li da %1$@ automatski provjerava nadogradnje? Nadogradnje možeš i ručno provjeriti u %1$@ izborniku.";
 
-"The update is improperly signed." = "Nadogradnja ima neispravnu signaturu.";
+"The update is improperly signed." = "Nadogradnja ima neispravan potpis.";
 
 "Update Error!" = "Greška prilikom aktualiziranja!";
 
-"Updating %@" = "Aktualiziranje %@";
+"Updating %@" = "Aktualiziranje %@a";
```
Comment (kornelski) [inline Sparkle/hr.lproj/Sparkle.strings:87]: That `a` doesn't seem right. If you're trying to give it a grammatical form, that may not work well with all app names, so it'd be better to find a different phrasing that doesn't require it.

File: Sparkle/hr.lproj/Sparkle.strings [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
+"%@ %@ is currently the newest version available.\n(You are currently running version %@.)" = "%1$@ %2$@ je trenutačno najnovija dostupna verzija.\n(Trenutačno pokrećeš verziju %3$@.)";
 
 /* Description text for SUUpdateAlert when the update is downloadable. */
-"%@ %@ is now available--you have %@. Would you like to download it now?" = "%1$@ %2$@ je sada dostupna. Ti koristiš inačicu %3$@. Želiš li je sada preuzeti?";
+"%@ %@ is now available--you have %@. Would you like to download it now?" = "%1$@ %2$@ je sada dostupna. Ti koristiš verziju %3$@. Želiš li je sada preuzeti?";
 
 /* Description text for SUUpdateAlert when the update informational with no download. */
-"%@ %@ is now available--you have %@. Would you like to learn more about this update on the web?" = "%1$@ %2$@ je sada dostupna. Ti koristiš inačicu %3$@. Želiš li saznati više o ovoj nadogradnji na webu?";
+"%@ %@ is now available--you have %@. Would you like to learn more about this update on the web?" = "%1$@ %2$@ je sada dostupna. Ti koristiš verziju %3$@. Želiš li saznati više o ovoj nadogradnji na internetu?";
```
Comment (kornelski) [inline Sparkle/hr.lproj/Sparkle.strings:17]: We mean Web specifically, not Internet here.

File: Sparkle/hr.lproj/Sparkle.strings (hunk 1 above)
Comment (milotype) [reply Sparkle/hr.lproj/Sparkle.strings:87]: Thanx, you're right, there is a better way. I'll change it.

File: Sparkle/hr.lproj/Sparkle.strings (hunk 2 above)
Comment (milotype) [reply Sparkle/hr.lproj/Sparkle.strings:17]: Ahh, OK, I'll change it.

Comment (milotype) [reply Sparkle/hr.lproj/Sparkle.strings:17]: Would the following interpretation be correct?
... learn more about this update on _our website_?

Comment (kornelski) [reply Sparkle/hr.lproj/Sparkle.strings]: Yes, that would be better. The English isn't specific about "our" part, but it's OK to add that if you need to.

**Comment by kornelski** [comment]
Thanks

**Pull request merged**